ci-operator/step-registry/ipi/conf/vsphere/vips/ipi-conf-vsphere-vips-commands.sh

```diff
@@ -12,8 +12,8 @@ fi
 
 third_octet=$(grep -oP '[ci|qe\-discon]-segment-\K[[:digit:]]+' <(echo "${LEASED_RESOURCE}"))
 
-echo "192.168.${third_octet}.2" >> "${SHARED_DIR}"/vips.txt
 echo "192.168.${third_octet}.3" >> "${SHARED_DIR}"/vips.txt
+echo "192.168.${third_octet}.4" >> "${SHARED_DIR}"/vips.txt
```

Comment:
@jianlinliu Here I removed 192.168.${third_octet}.2, because in IBMC devqe vSphere, `Do not use 192.168.*.1 or 2 for a static IP`, see doc https://docs.google.com/document/d/1cnzKMT-8TGcq5ox_AajGpT-VRwMuYTKIZI-fPNrR8Xg/. Do you have any suggestions? I'll ask dev to review it later, thanks.

Reply:
Ah, good point. I suggest to add one comment to indicate why .1 and .2 should not be used here.